src/components/toast/global_toast_list.test.tsx

```diff
@@ -186,5 +187,92 @@ describe('EuiGlobalToastList', () => {
         expect(dismissToastSpy).toBeCalled();
       });
     });
+
+    describe('clear all toasts button', () => {
+      const createMockToasts = (toastCount: number) =>
+        Array.from(new Array(toastCount)).map((_, idx) => ({
+          id: String(idx),
+          'data-test-subj': String(idx),
+        }));
+
+      const sharedProps = {
+        toastLifeTimeMs: 10,
+        dismissToast: jest.fn(),
+        toasts: createMockToasts(CLEAR_ALL_TOASTS_THRESHOLD_DEFAULT),
+      };
+
+      afterEach(() => {
+        sharedProps.dismissToast.mockClear();
+      });
+
+      test('is not visible when the showClearAllButtonAt prop is set to 0', () => {
+        const { queryByTestSubject } = render(
+          <EuiGlobalToastList
+            {...sharedProps}
+            showClearAllButtonAt={0}
+          />
+        );
```

Comment:
🤦 apologies for the spam
```suggestion
        const { queryByTestSubject } = render(
          <EuiGlobalToastList {...sharedProps} showClearAllButtonAt={0} />
        );
```